client/src/components/order/order.tsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from "react";
+import IOrder from "../../types/IOrder";
+import OrdersTable from "./ordersTable";
+
+const Order = () => {
```

Comment:
why creating another component if it is only using the `OrdersTable` component?

Reply:
Actually it's really unnecessary, I just wanted it to be like Item.